README.md

```diff
@@ -1475,7 +1481,8 @@ __Example:__
     let cart = await client.cart.create()
 
     const order = await client.checkout.shippingMethods({
-      orderToken: token.access_token
+      orderToken: token.access_token,
+      { include: 'shipping_rates' }
```

Comment:
Shouldn't it be `client.checkout.shippingMethods({orderToken: token.access_token}, {include: 'shipping_rates'})`?